opensearchpy/connection/krb_requests.py

```diff
@@ -0,0 +1,64 @@
+"""
+Class for the kerberos authentication.
+During the connection phase, one header is added with the kerberos context
+"""
+import warnings
+import kerberos
+
+from . import RequestsHttpConnection
+
+class RequestsKrbConnection(RequestsHttpConnection):
+    """ Class to connect with Kerberos """
+    def __init__(self,
+                 host="localhost",
+                 port=None,
+                 http_auth=None,
+                 use_ssl=False,
+                 verify_certs=True,
+                 ssl_show_warn=True,
+                 ca_certs=None,
+                 client_cert=None,
+                 client_key=None,
+                 headers=None,
+                 http_compress=None,
+                 cloud_id=None,
+                 api_key=None,
+                 opaque_id=None,
```

Comment:
Why is set of arguments different from the super class? Aren't we introducing defaults where we shouldn't?